src/index.ts

```diff
@@ -56,25 +58,22 @@ async function main(): Promise<void> {
 		}, r1);
 		const parts = r2.split('\n\n');
 
-		for (const part of parts) {
-			const body = {
-				avatar_url: process.env.WEBHOOK_AVATAR,
-				username: process.env.WEBHOOK_NAME,
-				content: part,
-			};
-
-			const res = await fetch(hook, {
-				method: 'post',
-				body: JSON.stringify(body),
-				headers: { 'Content-Type': 'application/json' },
-			});
-
-			if (res.status !== 204) {
-				console.error(await res.json());
-				throw new Error(`[API] ${res.status}: ${res.statusText}`);
+		let firstMessage: RESTPostAPIChannelMessageResult | null = null;
+		for (let part of parts) {
+			if (firstMessage) {
+				part = part.replace(
+					jumpRegex,
+					`https://discord.com/channels/222078108977594368/${firstMessage.channel_id}/${firstMessage.id}`,
+				);
 			}
-			await wait(1000);
```

Comment:
The wait was not there for rate limiting but because of a race condition sometimes resulting in the messages being sent in the wrong order. I'm not quite aware how that's supposed to be solved with the djs WebhookClient

Reply:
said race condition could have been because there the `wait` query parameter wasn't set to true in the URL, the djs WebhookClient sets it as true ([docs reference](https://discord.com/developers/docs/resources/webhook#execute-webhook))